frontend/pages/datasets/_workspace/_dataset/index.vue

```diff
@@ -45,6 +46,17 @@
       :datasetTask="dataset.task"
       @search-records="searchRecords"
     />
+    <base-modal
+      :modal-custom="true"
+      :prevent-body-scroll="true"
+      modal-class="modal-auto"
+      modal-position="modal-top-center"
+      :modal-visible="visibleTrainModal"
+      allow-close
+      @close-modal="showTrainModal(false)"
+    >
+      <DatasetTrainComponent :datasetTask="dataset.task" />
```

Comment:
see the comment about replacing some or all computed props by static variable => could be interesting to add a v-if on DatasetTrainComponent